src/lib/Backpack/devBackpack.cpp

```diff
@@ -191,6 +191,33 @@ static void AuxStateToMSPOut()
 #endif // USE_TX_BACKPACK
 }
 
+void crsfTelemToMSPOut(uint8_t *data)
+{
+    if (config.GetBackpackTlmEnabled() == 0)
+    {
+        // Backpack telem is off
+        return;
+    }
+    
+    mspPacket_t packet;
+    packet.reset();
+    packet.makeCommand();
+    packet.function = MSP_ELRS_BACKPACK_CRSF_TLM;
+
+    uint8_t size = CRSF_FRAME_SIZE(data[CRSF_TELEMETRY_LENGTH_INDEX]);
+    if (size > CRSF_MAX_PACKET_LEN)
+    {
+        ERRLN("CRSF frame exceeds max length");
```

Comment:
What happens if the size is too large and we send the packet anyway?

Should this actually be moved up a few lines and guard with a return?

Reply:
> What happens if the size is too large and we send the packet anyway?
The MSP buffer overruns, and the TX starts boot looping... ask me how I know :P

> Should this actually be moved up a few lines and guard with a return?
Yep